pkg/sql/planner.go

```diff
@@ -651,6 +651,11 @@ const (
 	KVStringOptRequireValue   KVStringOptValidate = `value`
 )
 
+// KVStringCaseInsensitiveOpts options which supports case Insensitive value
+var KVStringCaseInsensitiveOpts = map[string]struct{}{
+	"format": {},
+}
```

Comment:
We'll need to add more opts here, but this is the right idea ("whitelist" options we want to be case insensitive). I'll provide the full list here shortly.